src/status.router.ts

```diff
@@ -13,7 +14,7 @@ export type Status = {
 statusRouter.get('/', async (_req: Request, res: Response) => {
   const migrationStatus = (process.env.MIGRATION_STATUS ||
     'not_started') as MigrationStatus
-  if (!['not_started', 'in_progress', 'completed'].includes(migrationStatus)) {
```

Comment:
There was an inconsistency between `completed` and `complete`. I decided to go with complete because that is what the wallet app is expecting.